docs/glossary.md

```diff
@@ -221,6 +227,10 @@ A cryptographic function that produces a fixed-size hash value from variable-siz
 
 The number of hashes per second computed by miners on the network.
 
+#### HD Wallet
```

Comment:
Need to spell out acronym in header, i.e. 'Hierarchical Deterministic (HD) Wallet'